.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         make doc-spelling
     - name: Prepare twine checker
       run: |
-        pip install -U twine wheel
+        pip install --upgrade twine wheel
```

Comment:
`pip install` has [command line options](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-user) for:

    --upgrade
    --upgrade-strategy
    --user
    --use-pep517

In a workflow, it is better to be explicit about which of these four we are using.